src/UI/citizens/components/select-hall/CitizensSelectHall.js

```diff
@@ -2,53 +2,178 @@ import React from 'react';
 
-class CitizensSelectHall extends React.Component {
-  state = {
-    response: '',
-    error: ''
-  };
+// class CitizensSelectHall extends React.Component {
+//   state = {
+//     response: '',
+//     error: ''
+//   };
 
-  componentDidMount() {
-    this.fetchDataFromAPI();
-  }
+//   componentDidMount() {
+//     this.fetchDataFromAPI();
+//   }
 
-  fetchDataFromAPI = async () => {
-    try {
-      const apiRawData = await fetch(`${API_BASE_URL}/users`);
-      if (!apiRawData.ok) {
-        throw new Error('An error has occured!');
-      }
-      const apiJSONData = await apiRawData.text();
-      this.processResponse(apiJSONData);
-    } catch (error) {
-      this.processError(error.message);
-    }
-  };
-
-  processResponse = response => {
-    this.setState({
-      response,
-      error: ''
-    });
-  };
-
-  processError = error => {
-    this.setState({
-      response: '',
-      error
-    });
-  };
+//   fetchDataFromAPI = async () => {
+//     try {
+//       const apiRawData = await fetch(`${API_BASE_URL}/users`);
+//       if (!apiRawData.ok) {
+//         throw new Error('An error has occured!');
+//       }
+//       const apiJSONData = await apiRawData.text();
+//       this.processResponse(apiJSONData);
+//     } catch (error) {
+//       this.processError(error.message);
+//     }
+//   };
 
-  render() {
-    const { response, error } = this.state;
+//   processResponse = response => {
+//     this.setState({
+//       response,
+//       error: ''
+//     });
+//   };
+
+//   processError = error => {
+//     this.setState({
+//       response: '',
+//       error
+//     });
+//   };
+
+//   render() {
+//     const { response, error } = this.state;
 
+//     return (
+//       <div>
+//         <p>{response ? response : error}</p>
+//       </div>
+//     );
+//   }
+// }
+
+class CitizensSelectHall extends React.Component {
+  render() {
     return (
-      <div>
-        <p>{response ? response : error}</p>
-      </div>
+      <ScreenCanvas>
+        <MainContainer>
+          <Elipse>
+            <Number>1</Number>
+          </Elipse>
+          <LayoutWrapper>
+            <HeadlineText>Odaberite dvoranu</HeadlineText>
+            <SubheadlineText>
+              Pronađite dvoranu koja vam odgovara. Za dodatne informacije
+              kliknite na <TextInfoIcon src={Icon} /> ikonu.
+            </SubheadlineText>
+
+            <FirstRow>
+              <HallImage img={hallImage01}>
+                <a href="">
+                  <InfoIcon img={Icon} />
+                </a>
+                <HallNameText>
+                  <StyledLink to="">
+                    <HallNameTextCenter>
+                      Velika sportska dvorana
+                    </HallNameTextCenter>{' '}
+                  </StyledLink>
+                </HallNameText>
+              </HallImage>
+
+              <HallImage img={hallImage02}>
+                <a href="">
```

Comment:
Umjesto da 5 puta pišeš istu stvar, spremaj sve ove kolekcije komponenata u jedan array pa to renderiraj. Pogledaj kako je Mihael koristio taj isti pattern. Ili kako sam ja na nekim primjerima iz predavanja (lažni Trello)